_posts/0000-01-02-denise-erpenbach.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"When I am in the country," he replied, "I never wish to leave it; and when I am in town it is pretty much the same. They have each their advantages, and I can be equally happy in either."